aviary/api.py

```diff
@@ -96,7 +96,7 @@
 ###################
 
 # Miscellaneous
-from aviary.interface.methods_for_level2 import PreMissionGroup, PostMissionGroup
+# from aviary.interface.methods_for_level2 import PreMissionGroup, PostMissionGroup
```

Comment:
Since this is commented, I think you can just delete it. Anyone who needs these is doing pretty tightly integrated work, and would be safer pulling these from the source import instead of api.py anyway.